apacheconfig/parser.py

```diff
@@ -53,6 +59,8 @@ def p_includeoptional(self, p):
         """includeoptional : APACHEINCLUDEOPTIONAL
         """
         # lexer returns [includeoptional, whitespace, value]
+        if self._preserve_whitespace:
+            p[0] = ['includeoptional'] + p[1]
         p[0] = ['includeoptional', p[1][2]]
```

Comment:
Will this assignment override the previous one at line 63?

Reply:
Oh, great catch! This got messed up when I split up the larger PR. Added a test as well!